general-superstaq/general_superstaq/superstaq_client.py

```diff
@@ -498,7 +497,7 @@ def _prompt_accept_terms_of_use(self) -> None:
         response = self._accept_terms_of_use(user_input)
         print(response)
         if response != "Accepted. You can now continue using Superstaq.":
-            raise gss.SuperstaqException(
+            raise gss.SuperstaqServerException(
                 "You'll need to accept Terms of Use before usage of Superstaq.",
```

Comment:
nit (predates this pr)
```suggestion
                "You'll need to accept the Terms of Use before usage of Superstaq.",
```